src/test/java/roomescape/service/ThemeServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import roomescape.model.Theme;
+import roomescape.repository.fake.ReservationFakeRepository;
+import roomescape.repository.fake.ThemeFakeRepository;
+
+class ThemeServiceTest {
+
+    private ThemeService service;
+
+    @BeforeEach
+    void setUp() {
+        service = new ThemeService(new ReservationFakeRepository(), new ThemeFakeRepository());
+    }
```

Comment:
```suggestion
    private final ThemeService service = new ThemeService(new ReservationFakeRepository(), new ThemeFakeRepository());
```
junit5의 기본 라이프사이클은 메서드단위이므로 굳이 setUp을 따로 둘 필요는 없습니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle

Reply:
[`@Autowired`생략에 관한 코멘트](https://github.com/woowacourse/spring-roomescape-member/pull/243#discussion_r2072319737)와 같은 맥락으로 두었습니다.

setUp을 따로 두었을 때 차이가 없다면,
`@BeforeEach`를 작성함으로써 **각 테스트 메서드마다 service가 초기화됨을 명시하는 것이** 더 명시적이고 확실하지 않나요?

만약에 추후에 이 테스트의 라이프 사이클을 변경한다면 더 주의할 수 있지 않을까? 하는 생각도 듭니다.

Reply:
final이 없는데 "안전"하다고 하는 것이 맞을까요? 안전하려면 final을 다는 것이 더 안전할 것 같은데 service를 밑에서 변경할 수 있는 것 아닐까요?

명시적이다고 하셨는데 이미 junit5의 기본 테스트 라이프사이클 스펙이 메서드단위이므로 그것만으로도 초기화됨을 알 수 있을 것 같습니다.

테스트 라이프사이클이 변경된다면 그때 특정한 이유에 따라서 setUp을 커스텀할 수 있겠으나 현재는 라이프사이클이 변경될 상황이 만들어질 것 같지 않아요.

Reply:
말씀하신대로 `final 키워드와 함께` 필드에서 초기화하는 게 조금 더 안전할 것 같네요!